package.json

```diff
@@ -26,7 +26,9 @@
     "gulp-mocha-phantomjs": "^0.8.1",
     "mocha": "^2.2.5",
     "chai": "^3.2.0",
-    "gulp-jscs": "~2.0.0"
+    "gulp-jscs": "~2.0.0",
+    "requirejs-plugins": "^1.0.2",
```

Comment:
@GordonSmith can you add a package.json to https://github.com/GordonSmith/requirejs-plugins/tree/DOJO_AMD? npm will need it to install the package from a git repo.